docs/docusaurus.config.ts

```diff
@@ -165,6 +166,19 @@ const config: Config = {
 			},
 		],
 	],
+	headTags: [
+		{
+			tagName: "script",
+			attributes: {
+				src: "https://wcpstatic.microsoft.com/mscc/lib/v2/wcp-consent.js",
+				async: "true",
+			},
+		},
+	],
+	customFields: {
+		INSTRUMENTATION_KEY: process.env.INSTRUMENTATION_KEY,
```

Comment:
I'd like to see these variables documented. If I were doing so, I would:

1. Add a section to the README mentioning the website records some basic user telemetry
2. Add a .env.example file that is explicitly source controlled and contains the keys for all environment variables that are intended to be injected with some basic comments specifying where they come from (values are all fake, of course; use whatever makes that clear)

Reply:
There should already be an `.env.template` file for that purpose that you can modify.